src/helpers/figures/charts/runtime/chart_data_extractor.ts

```diff
@@ -490,8 +566,11 @@ function isLinearChart(
   return !definition.labelsAsText && canBeLinearChart(labelRange, getters);
 }
 
-export function canChartParseLabels(labelRange: Range | undefined, getters: Getters): boolean {
-  return canBeDateChart(labelRange, getters) || canBeLinearChart(labelRange, getters);
+export function canChartParseLabels(labelRange: Range[] | undefined, getters: Getters): boolean {
+  if (!labelRange || labelRange.length !== 1) {
+    return false;
+  }
+  return canBeDateChart(labelRange[0], getters) || canBeLinearChart(labelRange[0], getters);
```

Comment:
I think if there's multiple label ranges, we should say that there cannot be a linear/date axis. Taking the first one feels very arbitrary.

Reply:
This is exactly what is done with the if above

Reply:
Oh yeah I cannot read sorry